frontend/src/pulse/components/PulseEditChannels.jsx

```diff
@@ -183,14 +193,14 @@ export default class PulseEditChannels extends Component {
     renderChannelSection(channelSpec) {
         let { pulse, user } = this.props;
         let channels = pulse.channels
-            .map((c, index) => c.channel_type === channelSpec.type ? this.renderChannel(c, index, channelSpec) : null)
-            .filter(e => !!e);
+            .map((c, i) => [c, i]).filter(([c, i]) => c.enabled && c.channel_type === channelSpec.type)
```

Comment:
I'm not sure if this makes sense.  if the channels on a pulse are disabled then there is no way to access the pulse and edit it any more.